src/java/main/org/apache/zookeeper/server/quorum/QuorumPeer.java

```diff
@@ -1751,7 +1751,7 @@ public synchronized void initConfigInZKDatabase() {
         if (zkDb != null) zkDb.initConfigInZKDatabase(getQuorumVerifier());
     }
     
-    public void setRunning(boolean running) {
+    private void setRunning(boolean running) {
```

Comment:
I can't see the point of using a private setter. Set the field directly instead.

Reply:
I'm also ambiguous about this change.could you give me some your insight?

Reply:
Where are we with this @maoling @anmolnar ?

Reply:
Sorry for the late answer. It's not too much of an insight, rather - in general - I don't see the point of using a private setter without logic. In other words, is there any objection against setting the field directly?

Reply:
@maoling ?

Reply:
+1 to @anmolnar i think we can remote the setter